### DISCUSSION TITLE
I need changes to repo files in the container to be applied to local directory

### DISCUSSION THREAD

**Discussion by ktownsend-cbre** [nektos/act#2558] [Q&A] [answered]
I need changes made to repo files (i.e., new and updated in this case) that occur in my step to change my local folder, not just isolated in the container.

It is my understanding that ACT is copying my local files into the container and skipping the checkout step, which is perfect for my situation, but I also need changes from the run to make it back to my local folder so I can review the changes and optionally commit them. 

In my production run I am using a step `stefanzweifel/git-auto-commit-action@v5` to do this, but I have an `if` to skip it when running ACT because it doesn't work when the checkout action is skipped. I also need to review the changes and may not commit them. 

In my testing, it appears the changes are isolated to the container and don't make it back to my local folder. Is there a way to make that happen? I have not been able to find anything about it in my research.

**Comment by ChristopherHX** [comment, marked as answer]
You could try the dangerous`--bind` flag, this disables the copy and just mounts your folder. (and comes with known problems when you change the checkout directory in your workflow)

**Comment by ktownsend-cbre** [comment]
Thanks, that worked beautifully! I hadn't noticed that flag in the docs. Fortunately in my case I am not changing the checkout directory in my workflow.
